### PR TITLE
Add data license table

### LINKED ISSUE #1262: List Data Licenses
As reported by @karussell in #1235.

What do you think about a new section below the software for data licenses [here](https://github.com/graphhopper/graphhopper/blob/master/NOTICE.md) (and explicitly mention what is used by default)?

I would add stuff we have [here](https://graphhopper.com/api/1/docs/attribution/#data) and also gtfs.

Maybe via a table

|source | license | used as default | included in repo |
|---------|-----------|---------|------|
|OpenStreetMap data for the road network | [ODBL](https://www.openstreetmap.org/copyright) | yes | yes
| GTFS NRW | TODO link | no | yes
| SRTM elevation | [public domain](https://www2.jpl.nasa.gov/srtm/) | no | yes
| CGIAR elevation | [allowed usage for GraphHopper](https://graphhopper.com/public/license/CGIAR.txt) | no | yes
| SRTMGL1 elevation | TODO link | no | no
|OpenTopography mirror for SRTMGL1 | TODO  link | no | -  

This sounds like a reasonable thing to do and would make it easier for users to understand the used data and its implications.

### PR DESCRIPTION
**Pull request by boldtrn**
Have a look at the TODOs, I wasn't sure at 1 or 2 places.

Fixes #1262.

### CODE REVIEW

File: NOTICE.md [hunk 1]
```diff
@@ -51,3 +53,15 @@ android:
  * android (Apache license)
  * org.mapsforge, LGPL
  * VTM, LGPL
+
+## Data
+
+|source | license | used as default | included in repo |
+|---------|-----------|---------|------|
+|OpenStreetMap data for the road network | [ODBL](https://www.openstreetmap.org/copyright) | yes | yes
+| GTFS NRW | TODO link | no | yes (TODO - is it? I couldn't find it?)
```
Comment (karussell) [inline NOTICE.md]: Yes, I think @michaz was able to move everything in a sample file and did not need any other GTFS

File: NOTICE.md [hunk 2]
```diff
@@ -51,3 +53,15 @@ android:
...
+| SRTM elevation | [public domain](https://www2.jpl.nasa.gov/srtm/), [acknowledgement](https://lpdaac.usgs.gov/citing_our_data) | no | yes
+| CGIAR elevation | [allowed usage for GraphHopper](https://graphhopper.com/public/license/CGIAR.txt) | no | yes (TODO - really?, where?)
```
Comment (karussell) [inline NOTICE.md]: I thought in test, but then probably not :) Will have a look

Comment (karussell) [reply NOTICE.md]: We seem to use just SRTM data for tests under core/files

Comment (boldtrn) [reply NOTICE.md]: Yes, I think so too :).

File: NOTICE.md (hunk 1 above)
Comment (boldtrn) [reply NOTICE.md]: Ok, then I will keep it as yes?

Comment (boldtrn) [reply NOTICE.md]: What data is this? Is it from "Aachener Verkehrsverbund (AVV)" https://transit.land/feed-registry/operators/o-u-dkb?

Comment (boldtrn) [reply NOTICE.md]: If yes, then it seems to be open data (CC0). 

http://opendata.avv.de/
http://opendata.avv.de/lizenz_und_readme.txt

Comment (karussell) [reply NOTICE.md]: Yes it is open but we can remove it as just an artificial sample file is used. Sorry my wording wasn't clear enough.

Comment (boldtrn) [reply NOTICE.md]: No worries, I removed it :)

**Comment by karussell** [comment]
Thanks!

**Pull request merged**